examples/pennylane_run_rotoselect.py

```diff
@@ -161,11 +161,13 @@ def opt_theta(d, params, cost):
     a = np.arctan2(
         2.0 * M_0 - M_0_plus - M_0_minus, M_0_plus - M_0_minus
     )  # returns value in (-pi,pi]
+    print(a)
     params[d] = -np.pi / 2.0 - a
+    print(params[d])
     # restrict output to lie in (-pi,pi], a convention
```

Comment:
Just double checking that the above two print statements should be there!

Reply:
Yeah definitely not - should have made the docs before pushing.

Reply:
Okay all sorted